assets/components/subscribeCta/subscribeCta.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Heading, { type HeadingSize } from 'components/heading/heading';
+import CtaLink from 'components/ctaLink/ctaLink';
+import GridPicture from 'components/gridPicture/gridPicture';
+
+const BLOCK_NAME = 'component-subscribe-cta';
+
+export default function SubscribeCta() {
+  return (
+    <section className={BLOCK_NAME}>
```

Comment:
This was an attempt at being DRY... not especially fond of it tbh.

Reply:
I think when it comes to css class names it's best to just embrace the messiness, especially since they are replicated in the sass file anyway – we _could_ switch to css modules wink wink that would fix all of these issues